Core/include/Acts/Propagator/Propagator.hpp

```diff
@@ -64,28 +64,29 @@ struct PropagatorPlainOptions {
   /// Maximum number of steps for one propagate call
   unsigned int maxSteps = 1000;
 
-  /// Maximum number of Runge-Kutta steps for the stepper step call
-  unsigned int maxRungeKuttaStepTrials = 10000;
-
-  /// Absolute maximum step size
-  double maxStepSize = std::numeric_limits<double>::max();
-
   /// Absolute maximum path length
   double pathLimit = std::numeric_limits<double>::max();
 
-  /// Required tolerance to reach target (surface, pathlength)
-  double targetTolerance = s_onSurfaceTolerance;
+  /// Required tolerance to reach surface
+  double surfaceTolerance = s_onSurfaceTolerance;
```

Comment:
```suggestion
  double targetSurfaceTolerance = s_onSurfaceTolerance;
```
maybe?

Reply:
that tolerance is actually used for all the surfaces not just the target